_posts/2017-04-03-yarilets-critique.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title:  "Critique For U.S.News article on Chicago crime rate"
+date:   2017-04-10
+author: Yarilet Perez
+---
+<P>In a 2016 [article by U.S. News](https://www.usnews.com/news/articles/2016-09-19/chicago-drives-uptick-in-murders-national-crime-rate-stays-near-historic-lows) they state that although the national crime rate is at a “historic low,” the Chicago murder rate continues to rise. This comes from after information received from the Brennan Center for Justice, a policy institute at the New York University School of Law. The articles goes on to explain some of the reasons for the surge, and why the institute projects the rates will escalate on this subject when it comes to Chicago. They use sources from the Chicago Tribune, in which they report that the city has recorded 528 homicides in 2016.</P>
```

Comment:
You should remove these `<P>... </P>` tags. They aren't necessary when using Markdown, and though they are allowed, they'll mess up markdown inside your post. 

Also worth noting that modern HTML tags are always lowercase: `<p>Hello world.</p>`.